README.md

```diff
@@ -19,7 +19,10 @@ podman run -v /var/tmp/ai:/data:Z -it --rm quay.io/ohadlevy/ai-ipxe
 or let the container download the iso for you.
 
 ```
-podman run -e ISO_URL=http.... -v /var/tmp/ai:/data:Z -it --rm quay.io/ohadlevy/ai-ipxe 
+# ISO_URL is where the Assisted Installer ISO for your cluster is located
+# BASE_URL is where your iPXE files will be hosted
+
+podman run -e BASE_URL=$(hostname):8080 -e ISO_URL=http.... -v /var/tmp/ai:/data:Z --net=host -it --rm quay.io/ohadlevy/ai-ipxe 
```

Comment:
I didnt use --net=host on my env, why is it required?

Reply:
and while at it, can you fix the line above "this assumes that your image is located under /var/tmp/ipxe and its called image.iso" (it needs to be /var/tmp/ai)

Reply:
In my env didn't get any network and the ISO could not be reached